functions/Get-DbaSqlManagementObject.ps1

```diff
@@ -0,0 +1,88 @@
+﻿function Get-DbaSqlManagementObject
+{
+<#
+	.SYNOPSIS
+	Gets SQL Mangaement Object versions installed on the machine.
+
+	.DESCRIPTION
+	 The Get-DbaSqlManagementObject returns an object with the Version and the Add-Type Load Template for each version on the server.
```

Comment:
Just cosmetics, but would add a line break (make it 2 line sentence) in this to make it more readable in PowerShell console and such.